WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsActivity.kt

```diff
@@ -0,0 +1,74 @@
+package org.wordpress.android.ui.bloggingprompts
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.BloggingPromptsListActivityBinding
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.LocaleAwareActivity
+import org.wordpress.android.util.AppLog
+
+class BloggingPromptsActivity : LocaleAwareActivity() {
```

Comment:
I was wondering if we really need to inherit from `LocaleAwareActivity` in this case 🤔 
Do we use Locale somehow in this context?

Reply:
I've added a comment to remove the dependency on LocaleAwareActivity.